docs/setup_guides/cloud.rst

```diff
@@ -357,15 +196,7 @@ version of the Helm Chart, run the following commmand (if you are just updating
 jupyterhub authentication IP, ``--force`` and ``--recreate-pods`` are not
 needed).
 
-Scripts ``4_upgrade_helm.sh`` and ``5_upgrade_helm_soft.sh`` are available for
-that.
-
-.. code-block:: bash
-
-  $ helm upgrade --force --recreate-pods jupyter pangeo/pangeo \
-     --version=$VERSION \
-     -f secret_config.yaml -f jupyter_config.yaml
-
+The scripts '4_upgrade_helm.sh_' and '5_upgrade_helm_soft.sh_' are available for that.
```

Comment:
Afte some thoughts, maybe there is no need of scripts for one line command... Not sure about that.